UserDetailsClient/UserDetailsClient.Core/Features/LogOn/B2CAuthenticationService.cs

```diff
@@ -0,0 +1,141 @@
+﻿using Microsoft.Identity.Client;
+using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace UserDetailsClient.Core.Features.LogOn
```

Comment:
>UserDetailsClient.Core.Features.LogOn [](start = 10, length = 37)

nit: not sure I understand the namespace name

Reply:
using similar xamarin project organization that is found here...

https://github.com/Microsoft/TailwindTraders-Mobile/tree/master/Source/TailwindTraders.Mobile

i hope to expand this demo app to use the Shell control